package.json

```diff
@@ -19,6 +19,7 @@
     "test:focused": "tsx ./scripts/check-focused-tests.ts",
     "test:messages": "tsx ./scripts/check-messages.ts",
     "test:versions": "tsx ./scripts/check-versions.ts",
+    "test:markdown": "remark .",
```

Comment:
CI shows wrong warning

```
158:5-158:83 warning Cannot find file `(https:/github.blog/2019-02-14-introducing-draft-pull-requests/)` missing-file remark-validate-links:missing-file
```

Maybe it is a reason why PR is draft. Just note your.

Reply:
Also try to make a mistake to docs and run `pnpm test:markdown`. Script should fail with error (not just print warning).

Try to break HTTP link (remove any symbol to make it dead), break local files link, etc.